src/explorer.ts

```diff
@@ -188,6 +189,10 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
         this.resourceId = kind.abbreviation + '/' + id;
     }
 
+    namespace(): string | null {
+        return (this.metadata && this.metadata.namespace) ? this.metadata.namespace : null;
```

Comment:
is it even possible for an object to not have a namespace?

Reply:
It _is_ possible per the spec, though shouldn't happen on a get... but in any case `metadata` is optional and untyped so I don't want to presume anything.  I want to get it clearer and more consistent what the tree view nodes contain, and that might obviate this check, but for the time being I feel safer with it in!